### PR TITLE
New "Admin API and Stats API Access Token"

### PR DESCRIPTION
**Pull request by george-hoffer**

### CODE REVIEW

File: modules/ROOT/pages/mq-apis.adoc [hunk 1]
```diff
@@ -25,12 +25,11 @@ If you are a Windows user, you need to https://curl.haxx.se/download.html[downlo
 
 *Notes:* 
 
-* The access token can only be used with the Broker API. Do not use this access token 
-with the Stats and Admin APIs. For those APIs, you get the access token from the Anypoint
-Platform user.
-* The access token has a default time to live of 1 hour. This is the same TTL that users get when they log into the Anypoint Platform. The TTL is set in the Access Management screen for an organization in the Default Session Timeout value.
+* The access token in this section can only be used with the Broker API. Do not use this access token 
+with the Stats and Admin APIs. See <<Admin API and Stats API Access Token>> to get that access token.
+* The access token for any MQ API has a default time to live (TTL) of 180 minutes. This is the same TTL that users get when they log into the Anypoint Platform. The TTL is set in the Access Management screen for an organization in the Default Session Timeout value.
```
Comment (fededonna) [inline modules/ROOT/pages/mq-apis.adoc]: I'm not sure that access tokens for broker have a ttl of 180 minutes. Can you verify this with Authentication Server? Also ttl for those tokens can not be set in the ui.

File: modules/ROOT/pages/mq-apis.adoc [hunk 2]
```diff
@@ -25,12 +25,11 @@ If you are a Windows user, you need to https://curl.haxx.se/download.html[downlo
...
 
-To get API access information:
+To get the Broker API access token:
```
Comment (fededonna) [inline modules/ROOT/pages/mq-apis.adoc]: I'm not the best English speaker but I think "the" in front of the broker is not necessary.

File: modules/ROOT/pages/mq-apis.adoc [hunk 3]
```diff
@@ -51,6 +50,27 @@ This returns output similar to:
 {"access_token":"<token>","simple_client":{"envId":"<env_ID>","orgId":"<org-ID>"},"token_type":"bearer"}
 ----
 
+=== Admin API and Stats API Access Token
+
+For the Admin API and Stats API, get the access token from the Anypoint
```
Comment (fededonna) [inline modules/ROOT/pages/mq-apis.adoc]: same here

**Review by fededonna — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/mq-apis.adoc [hunk 4]
```diff
@@ -582,14 +510,118 @@ class AnypointMQAdminClient {
...
+      "created": "Tue, 23 Oct 2018 21:17:57 GMT",
+      "deliveryCount": "2"
+    },
+    ...
 ----
 
+After you have the lock ID, you can add it to your Postman environment to facilitate future requests.
+
+[[studiosetup]]
+=== Set Studio for API Access
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: setup?

File: modules/ROOT/pages/mq-apis.adoc [hunk 5]
```diff
@@ -64,125 +27,43 @@ Steps:
...
-== Use the Admin API
-
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-mq-admin/[Admin API] - Create and manage applications, queues, and message exchanges.
 
 In the Admin API portal, click Download to download the RAML or the API as a Mule Plugin.
 
 image::mq-rest-admin-portal.png[]
 
-== Create a FIFO Queue in the Admin API Portal
+=== Create a FIFO Queue in the Admin API
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: why did we get rid of portal? 
should be reverted or rephrased to **Create a FIFO Queue using Admin API**

File: modules/ROOT/pages/mq-apis.adoc [hunk 6]
```diff
@@ -208,25 +89,83 @@ Use this body:
...
++
+[source,json]
+----
+{"access_token":"<token>","simple_client":{"envId":"<env_ID>","orgId":"<org-ID>"},"token_type":"bearer"}
+----
 
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-mq-broker/[Broker API] - Publish, consume, ack, and nack messages between queues and message exchanges.
 
-In the Broker API portal, click Download to download the RAML or the API as a Mule Plugin.
+In the Broker API, click Download to download the RAML or the API as a Mule Plugin.
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: please add page or portal after Broker API.

**Review by jjayakumaran — CHANGES_REQUESTED** [review]

**Review by jjayakumaran — APPROVED** [review]

File: modules/ROOT/pages/mq-apis.adoc (hunk 2 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: The sentence is correct. The article "the" refers to "token".

File: modules/ROOT/pages/mq-apis.adoc (hunk 3 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Need to have the "the" article per "the ... API" and for "the ... token".

**Comment by fededonna** [comment]
LGTM

**Review by fededonna — APPROVED** [review]

**Pull request merged**